panther_power_control/src/relays_node.py

```diff
@@ -50,44 +63,44 @@ def __init__(self, name) -> None:
         #   Timers
         # -------------------------------
 
-        self._timer_set_motor = rospy.Timer(rospy.Duration(0.01), self._set_motor_state)
-        self._timer_e_stop = rospy.Timer(rospy.Duration(0.1), self._publish_e_stop_state)
+        # check motor state at 10 Hz
+        self._timer_set_motor = rospy.Timer(rospy.Duration(0.1), self._set_motor_state)
+
+        # init e-stop state
+        self._e_stop_state_pub.publish(self._e_stop_state)
 
         rospy.loginfo(f'[{rospy.get_name()}] Node started')
 
-    def _cmd_vel_cb(self, data) -> None:
-        self._cmd_vel_msg_last_time = time()
+    def _cmd_vel_cb(self, *args) -> None:
+        self._cmd_vel_msg_time = rospy.get_time()
 
     def _e_stop_reset_cb(self, req: TriggerRequest) -> TriggerResponse:
-        if time() - self._cmd_vel_msg_last_time <= 2.0:
+        if rospy.get_time() - self._cmd_vel_msg_time <= 2.0:
             return TriggerResponse(
                 False,
-                'E-STOP reset failed, some messages are published on the /cmd_vel topic',
+                'E-STOP reset failed, /cmd_vel is still being published!',
```

Comment:
This sentence sounds a bit weird, you don't publish topic rather messages to a topic. IMO previous was fine.

Reply:
Well. This is message copied from `power_board_node`

Reply:
Maybe we should change it in `power_board_node` too